.travis.yml

```diff
@@ -24,7 +24,10 @@ before_script:
 
   # Install composer dependencies
   - composer validate
-  - composer require --no-update silverstripe/recipe-core:4.6.x-dev silverstripe/versioned:1.6.x-dev
+  - composer require --no-update silverstripe/recipe-core:$RECIPE_VERSION
+  # silverstripe/versioned must be 1.x-dev (or 1.n.x-dev), not ^1
```

Comment:
Would be nice to clarify why exactly it *must* be `.x-dev` and not `^1`.
Are there some technical restrictions for that?
`"minimum-stability": "dev"` seems to be assigned in `composer.json`